packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -492,8 +504,12 @@ export class ExecutionEngineHttp implements IExecutionEngine {
       })
       .catch((e) => {
         if (e instanceof ErrorJsonRpcResponse && parseJsonRpcErrorCode(e.response.error.code) === "Method not found") {
-          this.lastGetBlobsErrorTime = timeNow;
-          this.logger.debug("disabling engine_getBlobsV1 api call since engine responded with method not availeble", {
+          if (isForkPostFulu(fork)) {
+            this.lastGetBlobsV2ErrorTime = timeNow;
```

Comment:
if we only call either getBlobs or getBlobsV2 then 2 variables can be merged, just drop a comment for it